easy_phi/static/js/main.js

```diff
@@ -53,24 +41,22 @@ var ep = window['ep'] || {
             $("#platform_info_sw_version").text(platform_info['sw_version']);
             $("#platform_info_hw_version").text(platform_info['hw_version']);
             $("#platform_info_slots").text(platform_info['slots']);
-            ep.updateModuleList(); // manually update list of modules
```

Comment:
module list update is not dependent on platform info anymore. Old design was creating containers for modules up to slots count and then dynamically added containers if new slot is added. The problem was race condition with websockets update, now all modules added dynamically by number of connected modules.